src/Swig/pxr/usd/usd/usdAttribute.i

```diff
@@ -57,4 +57,7 @@ typedef std::vector<UsdAttribute> UsdAttributeVector;
 %ignore UsdAttribute::GetTimeSamples(std::vector<double>* times) const;
 %ignore UsdAttribute::Get(VtValue* value) const;
 
+%ignore UsdAttribute::Set(const char* value) const;
+%ignore UsdAttribute::Set(const char* value, UsdTimeCode time = UsdTimeCode::Default()) const;
```

Comment:
Why did you need to ignore those two functions?

Reply:
Those 2 were added recently in usd as convenience overload and swig was complaining about ambiguous definition with
`
 UsdAttribute::Set(VtValue value) A
UsdAttribute::Set(VtValue value, UsdTimeCode time = UsdTimeCode::Default())
`